pipework

```diff
@@ -140,11 +136,11 @@ case "$N" in
   0)
     # If we didn't find anything, try to lookup the container with Docker.
     if installed docker; then
-      RETRIES=3
+      RETRIES=10
```

Comment:
Ah, let's not change this (at least, not in the same pull request, if possible...)